api/tests/Behat/bootstrap/v2/AppNotification/AppNotificationTrait.php

```diff
@@ -92,4 +92,20 @@ public function iShouldSeeAValidationError()
     {
         $this->assertOnErrorMessage($this->validationMsg);
     }
+
+    /**
+     * @Then I should see a banner confirming the :hostedEnvironment version of the app I am using
+     */
+    public function iShouldSeeABannerConfirmingTheVersionOfTheAppIAmUsing(string $hostedEnvironment)
+    {
+        $this->assertPageContainsText(sprintf('You are now logged into the %s environment', $hostedEnvironment));
+    }
+
+    /**
+     * @Then I should not see the banner confirming the version of the app I am using
+     */
+    public function iShouldNotSeeTheBannerConfirmingTheVersionOfTheAppIAmUsing()
+    {
+        $this->assertElementNotOnPage('govuk-notification-banner__heading');
```

Comment:
Its good to check for the banner itself but to make this less brittle (e.g. easy to break) we should probably assert the text doesn't appear anywhere on the page either - mainly because we could change the banner we use for this in the future and we wouldn't then never know if this test is actually passing.